src/libraries/System.Collections.Specialized/ref/System.Collections.Specialized.cs

```diff
@@ -219,6 +219,7 @@ public partial class StringCollection : System.Collections.ICollection, System.C
         void System.Collections.IList.Insert(int index, object? value) { }
         void System.Collections.IList.Remove(object? value) { }
     }
+    [System.ComponentModel.Design.Serialization.DesignerSerializerAttribute("System.Diagnostics.Design.StringDictionaryCodeDomSerializer, System.Design, Version=4.0.0.0, PublicKeyToken=b03f5f7f11d50a3a", "System.ComponentModel.Design.Serialization.CodeDomSerializer, System.Design, Version=4.0.0.0, PublicKeyToken=b03f5f7f11d50a3a")]
```

Comment:
[nit]
```suggestion
    [System.ComponentModel.Design.Serialization.DesignerSerializerAttribute("System.Diagnostics.Design.StringDictionaryCodeDomSerializer, System.Design, Version=4.0.0.0, PublicKeyToken=b03f5f7f11d50a3a", 
                                                                            "System.ComponentModel.Design.Serialization.CodeDomSerializer, System.Design, Version=4.0.0.0, PublicKeyToken=b03f5f7f11d50a3a")]
```

Reply:
These files are auto generated and need to match the format emitted by gen api tool